### PR TITLE
Add toValidation and toValidationNel to std.`try`

### PR DESCRIPTION
**Pull request by justjoheinz**

### CODE REVIEW

File: core/src/main/scala/scalaz/std/Try.scala [hunk 1]
```diff
@@ -26,6 +35,12 @@ trait TryInstances {
       def to[A](fa: Try[A]) = t.toDisjunction(fa)
       def from[A](ga: Throwable \/ A) = t.fromDisjunction(ga)
     }
+
+  val tryValidationIso: Try <~> λ[α => Validation[Throwable, α]] =
+      new IsoFunctorTemplate[Try, Validation[Throwable, ?]] {
+        def to[A](fa: Try[A]) = t.toValidation(fa)
+        def from[A](v: Validation[Throwable, A]) = t.fromValidation(v)
+      }
```
Comment (xuwei-k) [inline core/src/main/scala/scalaz/std/Try.scala]: could you fix the indenting?

**Comment by justjoheinz** [comment]
fixed indention.

**Comment by xuwei-k** [comment]
:+1:

**Comment by jedws** [comment]
:+1:

**Pull request merged**